TransformerModel.py

```diff
@@ -11,10 +11,11 @@
 # GNU General Public License for more details.
 
 from random import random
-
+import os
 import tensorflow as tf
 import tensorflow_hub as hub
 import  tensorflow_text as text
+os.environ['CUDA_VISIBLE_DEVICES'] = '-1'
```

Comment:
not sure if this is really needed . It might just run fine without it even if there is not GPU available.